jax/_src/scipy/special.py

```diff
@@ -599,7 +599,7 @@ def kl_div(
   .. math::
 
      \mathrm{kl\_div}(p, q) = \begin{cases}
-       p\log(p/q) & p>0,q>0\\
+       p\log(p/q)-p+q & p>0,q>0\\
```

Comment:
Please rebase on an updated main branch to exclude unrelated changes

Reply:
Done!